lib/jwt_signed_request/verify.rb

```diff
@@ -92,5 +93,25 @@ def verified_headers?
         Headers.fetch(header_key, request) == header_value
       end
     end
+
+    def parsed_claims_path
+      @parsed_claims_path ||= URI.parse(claims['path'])
+    end
+
+    def standard_query_values(path)
+      URI.decode_www_form(path.query).sort if path && path.query
+    end
+
+    def verified_query_strings?
+      claims_query_values == request_query_values
+    end
+
+    def claims_query_values
+      standard_query_values parsed_claims_path
```

Comment:
Should there be parentheses () around `parsed_claims_path`? It seems like parentheses are used for arguments elsewhere in the gem.